src/cli.rs

```diff
@@ -93,6 +96,7 @@ fn main() -> anyhow::Result<()> {
         verbose: cli.verbose,
         plain_text: cli.plain_text,
         export: cli.export,
```

Comment:
Actually, similar question for @angelayi , why did we end up needing this again? Shouldn't we be able to tell just from the structured log input whether there are export related logs?